modules/ROOT/pages/patterns/reference.adoc

```diff
@@ -0,0 +1,456 @@
+= Syntax and semantics
+
+This section contains reference material for looking up the syntax and semantics of specific elements of Graph Pattern Matching. 
+
+[[reference-node-pattern]]
+== Node pattern
+
+A node pattern is a pattern that matches a single node.
+It can be used on its own in a clause such as `MATCH` or `EXIST`, or form part of a xref:patterns/reference.adoc#reference-path-pattern[path pattern]. 
+
+See also xref:patterns/concepts.adoc#node-patterns[node pattern concepts].
+
+=== Syntax
+
+[source, syntax, role=noheader]
+----
+nodePattern ::= "(" [ nodeVariable ] [ labelExpression ] 
+    [ propertyKeyValueExpression ] [ "WHERE" booleanExpression ] ")"
```

Comment:
I thought this would be a simple fix, but that's not the case. It may have to be used with asciidoc 'callouts' - a kind of footnote system for source blocks. I will investigate a bit more to see if we can get a regular xref within the source block working.